src/dafny/Collections/Collections.md

```diff
@@ -0,0 +1,4 @@
+
+## The `Collections` module {#sec-collections}
+
+_Not yet written_
```

Comment:
We can safely delete this and fill in the gap later, right?

Reply:
We could. But this is clearer to people looking that the material is coming, rather than we just forgot about it.

Reply:
Ah fair enough, I thought this was meant as a TODO.